### DISCUSSION TITLE
Pouvoir réorganiser facilement une liste d'enrichissements

### DISCUSSION THREAD

**Discussion by camilledesalabert** [Inist-CNRS/lodex#2018] [Ideas]
Quand on doit ré-organiser une longue liste d'enrichissements (plus de 20), il serait plus simple de disposer d'un système de drag&drop

**Comment by AnaelKremer** [comment]
Petite remarque, s'il existe des dépendances entre les enrichissements cela peut vite être compliqué de s'y retrouver. Si le 5eme enrichissement s'appuie sur les données transformées dans le 2ème, pouvoir tout ré-organiser pourrait aussi tout désorganiser avec des enrichissemnts qui ne fonctionnent plus car reposant sur des champs pas encore existants.
Sensible donc.

**Comment by camilledesalabert** [comment]
Effectivement, mais c'est parfois l'occasion de remettre en ordre la liste d'un modèle existant que l'on a fait évoluer et qui nécessite au lancement de parfois le faire dans le désordre pour la raison que tu soulignes. Or les enrichissements en mode avancé ne permettent pas de connaître les dépendances sans les ouvrir tous un par un. Il faut donc mémoriser lequel intervient avec lequel => fastidieux.
